src/sentry/deletions/__init__.py

```diff
@@ -40,7 +40,6 @@ def load_defaults():
     default_manager.register(models.ApiToken, BulkModelDeletionTask)
     default_manager.register(models.CommitAuthor, BulkModelDeletionTask)
     default_manager.register(models.CommitFileChange, BulkModelDeletionTask)
-    # TODO(lb): Create a defaults.DashboardDeletionTask
```

Comment:
Do we actually need a `DashboardDeletionTask`? Or can we just rely on Django to follow all the links when we delete a Dashboard and just use `ModelDeletionTask`?

Reply:
No we don't need one. You're right.